### DISCUSSION TITLE
Unauthorized User

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5362] [Q&A] [answered]
After create user from cp and try to login with it via http://127.0.0.1:8000/cp/auth/unauthorized it gives me Unauthorized. i am using eloquent driver. Can i know the reason ?

**Comment by duncanmcclean** [comment, marked as answer]
Your user doesn't have access to the Control Panel. 

In the `users` table, there will be a `super` column (can't remember the exact name), make that `true`/`1` and go to `/cp` and you'll be able to see the CP.

**Reply by HazemMohamed252** [reply]
I know that super will access the cp but i am talking about normal user with limited roles and groups when trying to login it gives me Unauthorized

**Reply by duncanmcclean** [reply]
You'll need to create a role and then assign the user to the role then.

**Reply by HazemMohamed252** [reply]
I already created the role and assigned it to that user

**Reply by duncanmcclean** [reply]
Does that user have the ability to access the Control Panel? That'll be its own permission.

Can you share the role you've added to the user?

**Reply by HazemMohamed252** [reply]
Does i should grant him Access the Control Panel also beside any permissions ?

**Reply by duncanmcclean** [reply]
Yes, they'll need that to login to the CP without the error you're seeing.

**Reply by HazemMohamed252** [reply]
Thank so much
